tests/cypress/integration/actions_users/case_38_feedback_button.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Check feedback button.', () => {
+    const caseId = '38';
+
+    before(() => {
+        cy.visit('auth/login');
+        cy.login();
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Feedback button is available.', () => {
+            cy.get('.cvat-feedback-button').should('be.visible').click();
+        });
+
+        it('Feedback popover is available. Check content.', () => {
+            cy.get('.cvat-feedback-popover')
+                .should('be.visible')
+                .within(() => {
+                    cy.get('a').then(($a) => {
+                        const hrefValue = [];
+                        for (let i = 0; i < $a.length; i++) {
+                            hrefValue.push($a[i].href);
+                        }
```

Comment:
``const hrefValues = $a.map((a) => a.href)``

Reply:
In this case, an empty object is returned. Reworked without using a loop.